### ISSUE TITLE
ffmpeg writes empty image files (zero bytes) to NFS mounted on CAT

### ISSUE THREAD

**Issue by eduongAZ** [ml4ai/tomcat#253]
Images outputted by ffmpeg into NFS mount on CAT are empty: the files have zero byte. When outputting to a local folder, the images are valid and can be opened.

Could be related to #243

**Comment by eduongAZ** [comment]
ffmpeg crashes whenever outputting to NFS mounted drive. The error message seems unhelpful:

```
av_interleaved_write_frame(): Unknown error: 10038
Conversion failed!
```

**Comment by eduongAZ** [comment]
Problem fixed using #256 solution.
